heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -125,39 +142,23 @@ private static int runSyncProcess(
     // stream is read while waiting for the process to complete. If either buffer becomes full, it
```

Comment:
this comment talks about multiple buffers, which we no longer have below. please update to reflect new behaviour